site/content/contribute/team_contributions/_index.md

```diff
@@ -9,22 +9,44 @@ Mattermost is an open source Slack-alternative that helps teams stay in sync, wi
+
+## Solution Contributors
```

Comment:
What are `Solution Contributors`?

Reply:
Those who have created an integration, installer, package manager, plugin and more. Definitely open to other naming as well

Reply:
Maybe `Integration Contributors` as all these items are listed on https://integrations.mattermost.com/